pkg/registry/csv.go

```diff
@@ -53,6 +60,49 @@ type ClusterServiceVersion struct {
 	Spec json.RawMessage `json:"spec"`
 }
 
+func NewFromBundleDirectory(bundleDir string) (*ClusterServiceVersion, error) {
```

Comment:
Can we change this function name to be more descriptive? NewFromBundleDirectory describes the mechanics of what the function is doing, not the purpose or the output. Maybe `ReadCSVFromBundleDirectory`?